README.md

```diff
@@ -65,6 +65,15 @@ it code.
 }
 ```
 
+## Samples
+
+With the power of Node.js and npm, the sky is the limit. What are you going to build today?
+
+- [/fusebot kudos](samples/kudos) - leave kudos for your team members, check ranking, and more
+- [/fusebot coin](samples/coin) - get current value of a digital currency
+- [/fusebot run-query](samples/dataQuery) - run SQL queries against MySQL compatible datatabases
+- [/fusebot github-dispatch](samples/githubDispatch) - trigger a github action
```

Comment:
I moved the section around, can you add the listing here?

Reply:
Done